qa/tasks/cephfs/test_misc.py

```diff
@@ -504,6 +504,12 @@ def test_dump_inmemory_log_on_missed_internal_heartbeats(self):
                 return
         self.assertTrue(False, "Failed to dump in-memory logs during missed internal heartbeat")
 
+    def test_session_ls_with_filters(self):
+        mount_a_client_id = self.mount_a.get_global_id()
+        info = self.fs.mds_asok(['session', 'ls', "%s" % mount_a_client_id])
```

Comment:
```suggestion
        info = self.fs.mds_asok(['session', 'ls', f"id={mount_a_client_id}"])
```

Reply:
Add the same test for `client ls`. Test `cap_dump` flags too. Test "vanilla" (no argument) command too.